client/packages/system/src/Stock/Components/StockLineForm.tsx

```diff
@@ -90,173 +104,182 @@ export const StockLineForm: FC<StockLineFormProps> = ({
     return () => document.body.removeEventListener('keydown', handleKeyDown);
   }, []);
 
+  if (loading) return null;
+
   return (
-    <Grid
-      display="flex"
-      flex={1}
-      container
-      paddingTop={2}
-      paddingBottom={1}
-      width="100%"
-      flexWrap="nowrap"
-    >
+    <DetailContainer>
       <Grid
-        container
         display="flex"
         flex={1}
-        flexBasis="50%"
-        flexDirection="column"
-        gap={1}
+        container
+        paddingTop={2}
+        paddingBottom={1}
+        width="100%"
+        flexWrap="nowrap"
+        maxWidth={900}
+        gap={footerProps ? 10 : undefined}
```

Comment:
This component is displayed in both a modal (when creating new) and on the page, so requires different spacing depending on which it is.

Reply:
Are we happy with the Modal for "Create new", or should we launch straight into an empty page? (I think this modal is fine, personally)

Reply:
I think the modal is fine, its pretty consistent with the create patterns we have elsewhere. Might defer to @mark-prins on that though?